assets/stylesheets/environment/tools/_placeholders.scss

```diff
@@ -5,3 +5,9 @@
     display: block;
   }
 }
+
+// Block for a layout "row" - i.e. header, content, footer
+%content-block {
```

Comment:
Should this sort of thing be part of a grid "object"?

And how are we planning to build the grid - using semantic class names or using mixins and placeholders added to components?